assets/app.scss

```diff
@@ -0,0 +1,114 @@
+@import '../node_modules/wikimedia-ui-base/wikimedia-ui-base.css';
+@import '../vendor/wikimedia/toolforge-bundle/Resources/assets/wikimedia-base.css';
+
+:root {
+    // Workaround for https://github.com/webpack-contrib/css-loader/issues/708
+    border: 0;
+}
+
+html,
+body {
+    height: 100%;
+}
+
+body {
+    background-color: var( --wmui-color-base80 );
+    font-size: 0.9rem;
+}
+
+header,
+footer {
+    max-width: var(--width-breakpoint-tablet);
+    margin: 3rem auto;
+}
+
+header {
+    h1 {
+        font-size: 2.25rem;
+        font-weight: 600;
+        letter-spacing: 1px;
+        font-family: var( --font-family-base );
+    }
+    nav.user {
+        float: right;
+        text-align: right;
+    }
+}
+
+footer {
+    .toolforge {
+        float: left;
+    }
+    .toolforge-logo {
+        -webkit-filter: grayscale(100%);
+        filter: grayscale(100%);
+    }
+    .toolforge-logo:hover {
+        -webkit-filter: none;
+        filter: none;
+    }
+    .info {
+        text-align: right;
+    }
+}
+
+main,
+aside.tutorial {
+    margin: 1.5rem auto;
+    max-width: var( --width-breakpoint-desktop-wide );
+    padding: 1rem;
+}
+
+main {
+    background-color: var( --background-color-base );
+}
+
+aside.tutorial {
+    border-radius: var( --border-radius-base );
+    border: 1px solid var( --border-color-base );
+    ol {
+        max-width: var(--width-breakpoint-desktop);
+        list-style-type: none;
+        display: flex;
+        margin: auto;
+    }
+    li {
+        font-size: 1rem;
+        flex: 1;
+        display: flex;
+        align-items: center;
+    }
+    p.image {
+        flex: 1;
```

Comment:
Flex is actually only partially supported in Edge (see https://caniuse.com/#search=flex )
We avoid using it in MW, but the target audience of this product is different...  We should make sure we test this in IE10/11 and see if we need to redo the approach.

Reply:
I thought flex was okay, it was just css grid that we're avoiding. But you're right, there's likely to be problems. (Side note: we're using flex in templatewizard!)

I don't have access to edge. Should I come up with a float-only way of doing this same layout? Or should we just stick with bootstrap's grid (we can include just it)?